app/web/src/store/asset.store.ts

```diff
@@ -35,20 +35,20 @@ export interface InstalledPkgAssetView {
 
 export type DetachedAttributePrototypeKind =
   | {
-      type: "ExternalProviderSocket";
+      type: "OutputSocketSocket";
       data: {
         name: string;
         kind: "ConfigurationInput" | "ConfigurationOutput";
       };
     }
   | {
-      type: "InternalProviderSocket";
+      type: "InputSocketSocket";
```

Comment:
These sound and look bad. It's because they are! This is part of renaming and unblocking folks from working on func authoring. We'll come back as we cull some of the old logic here.